core/src/main/java/org/jruby/parser/ParserSupport.java

```diff
@@ -1453,13 +1453,8 @@ public Node arg_append(Node node1, Node node2) {
 
     // MRI: reg_fragment_check
     public void regexpFragmentCheck(RegexpNode end, ByteList value) {
```

Comment:
Since 9.2 will be considered a major version you can remove this from ParserSupport if you want (or not) and just make callers call lexer directly.  Optional.

Reply:
Done